src/components/Preview.vue

```diff
@@ -36,7 +36,16 @@
         wrap="hard"
     />
     <div class="validation-msg">
-        <p v-bind:class="isValidCFF ? '' : 'invalid'">
+        <p
+            v-if="isMinimumCFF"
+            v-bind:class="isValidCFF ? '' : 'invalid'"
```

Comment:
```suggestion
            class="invalid"
```
Because isMinimumCFF => !isValidCFF.
By the way, the name should be more descriptive of the check. Something like `isFailingMinimumCFF`, for instance.